### PR TITLE
Deleted unecessay files

### PR DESCRIPTION
**Pull request by Snowflake8**

### CODE REVIEW

File: app/src/main/java/com/github/polybooks/AddSaleActivity.kt [hunk 1]
```diff
@@ -10,9 +10,9 @@ import androidx.appcompat.app.AppCompatActivity
 const val ISBN = "com.github.polybooks.ISBN"
 
 
-/* This activity is the starting point to add a new sale. It gives the option between scanning the book barcode or manually inputting the ISBN.
+/** This activity is the starting point to add a new sale. It gives the option between scanning the book barcode or manually inputting the ISBN.
```
Comment (Huntsekkern) [inline app/src/main/java/com/github/polybooks/AddSaleActivity.kt:13]: What's the difference between those two styles of commenting?

**Review by Huntsekkern — APPROVED** [review]
As discussed orally, all good with the intention, and it seems you're already fixing the issue the CI raised

File: app/src/main/java/com/github/polybooks/AddSaleActivity.kt (hunk 1 above)
Comment (Snowflake8) [reply app/src/main/java/com/github/polybooks/AddSaleActivity.kt:13]: This marks it as javadoc and not just comments, meaning if you hover over the class (from another file) this description will appear

Comment (Huntsekkern) [reply app/src/main/java/com/github/polybooks/AddSaleActivity.kt:13]: Ok, thanks for the explanation and for already fixing it!

**Comment by simhayoz** [comment]
@Snowflake8 Could you please not ask everyone for reviews? This makes a lot of unnecessary notification. Moreover, it is better to ask 2 or 3 reviewers in a meaningful way, for instance asking people that have already worked with the same part of the code base or people that may have an interest for it later (or people that have not done much reviews yet).

Thanks :smile:

**Review by DGoedtkindtEPFL — APPROVED** [review]
This is just deleting unused files, I have nothing to say against it

**Pull request merged**